composer.json

```diff
@@ -18,16 +18,19 @@
         "forum": "https://discourse.laminas.dev"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "platform": {
+            "php": "8.0.99"
+        },
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
         "laminas/laminas-view": "^2.8.1",
         "laminas/laminas-zendframework-bridge": "^1.0"
     },
-    "require-dev": {
-        "laminas/laminas-console": "^2.6"
```

Comment:
laminas-console is still in use:

https://github.com/laminas-api-tools/api-tools-admin-ui/blob/e77c5b1295400d8695f26c484a338bafdf1668ab/bin/ui-mode.php#L4-L11

Reply:
Hi @froschdesign, thank you for your review.

If it is used, why is it on `require-dev`?  Is it ok if I move it to the `require` section?

Reply:
I don't know, but it will not work anyway:

* https://github.com/laminas/laminas-console/blob/bffaf29cf357a87b5266b6cc5f4e137b83ffe52e/composer.json#L28
* https://github.com/laminas/laminas-console#laminas-console

Reply:
Then I will keep it as it was and open another PR to replace laminas-console with laminas-cli